docs/shared-content-source/docs/modules/ROOT/pages/app-development-process.adoc

```diff
@@ -42,9 +56,10 @@ include::{cloudflow-examples-version}@docsnippets:ROOT:example$spark-scala/src/m
 
 Akka Streams and Flink-based `Streamlets` follow the same pattern. 
 
-== Compose application flow with a blueprint
+== Compose The Application Flow with a Blueprint
```

Comment:
Very minor but I thought `The` shouldn't be capitalized unless it's at the start of a title